packages/relay/src/lib/services/hbarLimitService/index.ts

```diff
@@ -359,6 +378,14 @@ export class HbarLimitService implements IHbarLimitService {
     Object.values(SubscriptionTier).forEach((tier) => this.uniqueSpendingPlansCounter[tier].reset());
   }
 
+  /**
+   * Resets the number of active users per rate limiter duration.
+   * @private
+   */
+  private resetActiveUsers(): void {
+    this.activeUsersPerLimitPeriodCounter.reset();
```

Comment:
I don't think we should be resetting in code. 
Shouldn't we be able to group this within the metric collection tool e.g. Grafana
For instance if the duration is not 24hrs the intent breaks